src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3638,9 +3638,10 @@ void MethodContext::recGetThreadLocalStaticBlocksInfo(CORINFO_THREAD_STATIC_BLOC
 
     value.tlsIndex.handle                   = CastHandle(pInfo->tlsIndex.addr);
     value.tlsIndex.accessType               = pInfo->tlsIndex.accessType;
-    value.offsetOfMaxThreadStaticBlocks     = pInfo->offsetOfMaxThreadStaticBlocks;
-    value.offsetOfThreadLocalStoragePointer = pInfo->offsetOfThreadLocalStoragePointer;
-    value.offsetOfThreadStaticBlocks        = pInfo->offsetOfThreadStaticBlocks;
+    value.offsetOfMaxThreadStaticBlocks        = pInfo->offsetOfMaxThreadStaticBlocks;
+    value.offsetOfThreadLocalStoragePointer      = pInfo->offsetOfThreadLocalStoragePointer;
+    value.offsetOfThreadStaticBlocks           = pInfo->offsetOfThreadStaticBlocks;
+    value.offsetOfGCDataPointer                  = pInfo->offsetOfGCDataPointer;
```

Comment:
Can you fix the alignment?

Reply:
seems these projects are excluded from auto-fixing using formatter?

Reply:
Sergey set up using the formatter, but it's not part of the normal JIT runs or validation in CI.